resources/translations/en.edn

```diff
@@ -185,6 +188,7 @@
              :show "Show"
              :show-less "Show less"
              :show-more "Show more"}
+  :change-form {:form-selection "Form"}
```

Comment:
Should more translation keys be in this map? At least the intro text.

Reply:
I don't see the need for premature duplication of the translation texts. We don't intend to normalize / dedupe them anyway for other things.

Reply:
I mean that `:t.administration/change-form-intro` is not a generic text used on many administration page, but specific to the change-form page, so a renaming it to `:t.change-form/intro` would be more cohesive.

Reply:
Likewise `:t.administration/change-form` could be renamed to `:t.change-form/title`

Reply:
I will move the intro, that text is actually leftover from earlier time when it was still on the same page. I think we often use the more general texts in the specific sub-pages than vice versa so I think I will leave the latter to admin.